sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -41,6 +41,22 @@ public partial class ClientResult<T> : System.ClientModel.ClientResult
         public static implicit operator T (System.ClientModel.ClientResult<T> result) { throw null; }
     }
 }
+namespace System.ClientModel.Options
+{
+    public partial class LoggingOptions
+    {
+        public LoggingOptions() { }
+        public bool IsLoggingContentEnabled { get { throw null; } set { } }
+        public bool IsLoggingEnabled { get { throw null; } set { } }
+        public string? LoggedClientAssemblyName { get { throw null; } set { } }
+        public int LoggedContentSizeLimit { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> LoggedHeaderNames { get { throw null; } set { } }
```

Comment:
Wondering if maybe calling these "Allowed" would make it more clear that this feature is about security?  But that is an architect call, ultimately.

Also, wondering if the plan is to specify allowed headers and query parameters for logging and distributed tracing policies separately, or would there be a central place that would govern those values for the entire client?

Reply:
Yeah I like "Allowed" more too, will update

In Core they're shared, but I'm thinking in SCM they will be separate. In Core, the diagnostics options are shared by all the different diagnostics related policies. In SCM though it seems like it aligns more with the environment and goals to have logging + logging options, distributed tracing + distributed tracing options, etc.

Reply:
We could still do that tho I guess but put the allow lists on the ClientPipelineOptions, I feel like it's probably more usable to have all the options for each policy in one place, but I'll think a bit more and come back to this